src/redux/reducers/tweets.ts

```diff
@@ -231,6 +237,20 @@ const fetchMentions = createAsyncThunk<Pagination<Tweet2>, FetchMentionsArgs>(
   }
 );
 
+interface TweetLike {
+  user: string;
+  tweet: string;
+  _id: string;
+}
+
+export const likeDislike = createAsyncThunk<
+  TweetLike,
+  { tweet: string | undefined }
```

Comment:
```suggestion
  { tweet: string }
```
what do we want to like if tweet = undefined? How can it be?

Reply:
```suggestion
  { tweet?: string }
```
if some way tweet can be undefined